src/components/related_items_outfit/YourOutfit.jsx

```diff
@@ -25,22 +22,21 @@ export default function YourOutfit({ curProd }) {
           flexDirection: 'row',
         }}
       >
-        {/* Conditional render if there are no products on Outfit */}
         <AddOutfitCard handleAdd={handleAdd} />
         <div>
-          <Carousel>
+          <Carousel outfit={outfit}>
             {outfit.map((product) => (
               <ProductCard product={product} setOpenModal={setOpenModal} />
```

Comment:
Just seeing now that there's no key here - can you add one rq?

Also seeing that this is missing the image prop. Can you pass a dummy image here for now to keep the console clear of warnings?

Reply:
Yeah, I think I pushed too early before a commit. Those are fixed.